src/GitTfs/Core/GitTfsRemote.cs

```diff
@@ -37,8 +37,10 @@ public class GitTfsRemote : IGitTfsRemote
             Id = info.Id;
             TfsUrl = info.Url;
             TfsRepositoryPath = info.Repository;
-            TfsUsername = info.Username;
-            TfsPassword = info.Password;
+            TfsUsername = info.Username ?? _remoteOptions.Username;
+            TfsPassword = info.Password ?? _remoteOptions.Password;
+            TfsPAT = info.PAT ?? _remoteOptions.PAT;
+
```

Comment:
Hm. Isn't the `RemoteInfo info` object what is stored in the .git/config whereas the  `remoteOptions` capture the
commandline arguments?

If my assumption is correct, this would mean that any user supplied user, password or PAT on the commandline
will be overwritten with what was stored in the .git/config file.
The other way around is what is preferfed here,  an explicit given user, password or PAT on the commandline should overwrite what was specified in the config.

On second thought, that would mean that prior to your change, specifying a User or Password from the commandline would also not work, as always the version from the config would be taken.

Reply:
They will if they are null. since I coalesced them. I found it odd that the username/ password weren't coming from the command line when run from the command line.

Reply:
Ok, then I would prefere that we change the behavior to use the options from the commandline if given, like you suggested with 
> ... the coalesce should be reversed